test/intergration/search/index_page.spec.js

```diff
@@ -0,0 +1,26 @@
+const assert = require('assert');
+const shunterTestHelper = require('shunter').testhelper();
+const fixtureHelper = require('../../helpers/fixture-helper');
+const paths = require('../../helpers/walk-helper');
+
+const b = require('js-beautify').html;
+
+describe('Search index page', function(){
+  before(function() {
```

Comment:
If these two methods are common functionality - can this be moved into a helper?

Reply:
Also, favour `beforeEach` and `afterEach`